invenio_app_ils/document_requests/api.py

```diff
@@ -105,7 +105,7 @@ class DocumentRequest(IlsRecord):
     """DocumentRequest record class."""
 
     STATES = ["ACCEPTED", "PENDING", "REJECTED"]
-    REJECT_TYPES = ["USER_CANCEL", "IN_CATALOG", "NOT_FOUND"]
+    REJECT_TYPES = ["USER_CANCEL", "IN_CATALOG", "NOT_FOUND", "MIGRATED"]
```

Comment:
I think that we should not add stuff about migration in `invenio-app-ils` if possible... can you change it to `OTHER` or something else?